caikit/core/data_model/streams/data_stream.py

```diff
@@ -38,13 +39,14 @@
 from ...augmentors import AugmentorBase
 from ...exceptions import error_handler
 from ...toolkit import fileio
+from ..json_dict import JsonDictValue
+from .multipart_decoder import is_multipart_file, stream_multipart_file
```

Comment:
NIT: I _think_ there's some dark corner of `toolkit` that has a bunch of `is_xxxxx` functions. Should `is_multipart_form` live there?

Reply:
There was a `isa` file of functions in a legacy version of this codebase but I don't see it (or those functions) existent in this codebase anymore.